controllers/cloud.redhat.com/config/schema.json

```diff
@@ -274,6 +277,25 @@
                 "tls"
             ]
         },
+        "FeatureFlagsConfig": {
```

Comment:
Is there no credential needed to query the feature flag server?

Reply:
Not currently